### PR TITLE
Fix broadcaster usage for transaction - Closes #3588

### LINKED ISSUE #3588: Transactions are not broadcasted after received by postTransactions
### Expected behavior
Transactions should be broadcasted to rest of the network after received by API.
### Actual behavior
They are not.
### Steps to reproduce
Send the transaction to a node that has forging disabled, check for its presence on other nodes.
### Which version(s) does this affect? (Environment, OS, etc...)
`2.0.0-alpha.0`

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?
Syntax of the awaiting the promise was wrong, and it was to parsed before queuing

### How did I fix it?
parse to normal object before queueing and use correct syntax for waiting promise

### How to test it?
run 2 nodes and send a transaction

### Review checklist

* The PR resolves #3588 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: framework/src/modules/chain/logic/broadcaster.js [hunk 1]
```diff
@@ -253,13 +253,11 @@ class Broadcaster {
...
 			const squashedBroadcasts = this.squashQueue(broadcasts);
 
-			// eslint-disable-next-line no-restricted-syntax
-			for await (const squashedBroadcast of squashedBroadcasts) {
-				return this.broadcast(
-					squashedBroadcast.params,
-					squashedBroadcast.options
-				);
-			}
+			await Promise.all(
```
Comment (diego-G) [inline framework/src/modules/chain/logic/broadcaster.js]: This is not necessary.

File: framework/src/modules/chain/submodules/transport.js [hunk 2]
```diff
@@ -306,16 +306,17 @@ Transport.prototype.onUnconfirmedTransaction = function(
...
 ) {
 	if (broadcast && !__private.broadcaster.maxRelays(transaction)) {
+		const transactionJSON = transaction.toJSON();
 		__private.broadcaster.enqueue(
 			{},
 			{
 				api: 'postTransactions',
 				data: {
-					transaction,
+					transaction: transactionJSON,
```
Comment (diego-G) [inline framework/src/modules/chain/submodules/transport.js]: TBH, I'm not so sure what we are fixing here. Anyways, in the case we do fix something, we should perform the same action in other places like in here https://github.com/LiskHQ/lisk-sdk/blob/61969851e484a18e119810851177f85091c3e774/framework/src/modules/chain/submodules/transport.js#L288

Comment (shuse2) [reply framework/src/modules/chain/submodules/transport.js]: signature is always an object so it should not need the toJSON

File: framework/src/modules/chain/logic/broadcaster.js (hunk 1 above)
Comment (shuse2) [reply framework/src/modules/chain/logic/broadcaster.js]: the usage of the `for await` should be for async generator or async iterables

**Pull request merged**